apps/admin-x-activitypub/src/api/activitypub.ts

```diff
@@ -407,12 +376,12 @@ export class ActivityPubAPI {
         return this.getPaginatedPosts('.ghost/activitypub/inbox', next);
     }
 
-    async getPostsByAccount(next?: string): Promise<PaginatedPostsResponse> {
-        return this.getPaginatedPosts('.ghost/activitypub/posts', next);
+    async getPostsByAccount(handle: string, next?: string): Promise<PaginatedPostsResponse | GetProfilePostsResponse> {
+        return this.getPaginatedPosts(`.ghost/activitypub/posts/${handle}`, next);
     }
 
-    async getPostsLikedByAccount(next?: string): Promise<PaginatedPostsResponse> {
-        return this.getPaginatedPosts('.ghost/activitypub/posts/liked', next);
+    async getPostsLikedByAccount(handle: string, next?: string): Promise<PaginatedPostsResponse> {
+        return this.getPaginatedPosts(`.ghost/activitypub/posts/liked/${handle}`, next);
```

Comment:
Had to add the handle here as `posts/liked` was otherwise routing to `posts/:handle`

Reply:
What about `/posts/me/liked`?

Reply:
Yeah, good suggestion. Done!